crates/aptos-crypto/benches/bls12381.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_use]
+extern crate criterion;
+
+use criterion::{
+    measurement::Measurement, BatchSize, BenchmarkGroup, BenchmarkId, Criterion, Throughput,
+};
+use rand::{distributions::Alphanumeric, rngs::ThreadRng, thread_rng, Rng};
+
+use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
+use serde::{Deserialize, Serialize};
+
+use aptos_crypto::{
+    bls12381,
+    bls12381::ProofOfPossession,
+    test_utils::{random_keypairs, random_subset, KeyPair},
+    traits::{Signature, SigningKey, Uniform},
+};
+
+#[derive(Debug, CryptoHasher, BCSCryptoHash, Serialize, Deserialize)]
+struct AptosBenchCrypto(String);
```

Comment:
probably should unify it with `TestAptosCrypto` and maybe reuse the existing one with right cfg!(bench)